### PR TITLE
kinesis changes + smaller misc modsuit stuff

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
renames plasmaman helmet exempt flag to stackable helmet exempt. hat stabilizer now cares about this instead of letting you stack eva helmets on top of modsuit helmets
improves some code that dripped in quality when i was on break
kinesis has some tweaks
previously it used to stun for a set amount of time - i think this is because the creator couldnt figure out how to stop movement while being held? this is changed, now as long as youre holding someone they are immobilized
you can cancel your kinesis without throwing or moving out of range by using right click on the click catcher
adds an admin version of kinesis just for the admin suit. it can grab (almost) ANYTHING and config menu lets you enable phasing, which makes it so the atom you grabbed phases through everything. pick up that can.

## Why It's Good For The Game
its fun

## Changelog
:cl:
balance: hat stabilizer module can now hold what plasmaman helmets can hold
qol: kinesis module can be stopped without launching an object with right click
balance: kinesis module stuns last until the kinesis stops
add: admin suit has a version of kinesis that can pick up anything at any range and can be configured to make grabbed mobs phase through walls :)
/:cl:

### CODE REVIEW

File: code/modules/mod/modules/module_pathfinder.dm [hunk 1]
```diff
@@ -21,7 +21,13 @@
...
 
 /obj/item/mod/module/pathfinder/Destroy()
-	implant = null
+	QDEL_NULL(implant)
+	return ..()
+
+/obj/item/mod/module/pathfinder/Exited(atom/movable/gone, direction)
+	if(gone == implant)
+		implant = null
+		icon_state = "pathfinder_empty"
```
Comment (MrMelbert) [inline code/modules/mod/modules/module_pathfinder.dm:30]: Should put the icon state part in `update_icon_state`

Comment (Fikou) [reply code/modules/mod/modules/module_pathfinder.dm:30]: smart thinking smellbert

**Comment by carlarctg** [comment]
PLEASE make a kinesis gravity gun

File: code/modules/mod/modules/module_kinesis.dm [hunk 2]
```diff
@@ -215,25 +246,87 @@
...
 		The bane of all ne'er-do-wells, the kinesis+ module is a powerful tool that allows the user \
 		to manipulate the world around them. Like it's older counterpart, it's capable of manipulating \
-		structures, machinery, vehicles, and, thanks to the fruitful efforts of it's creators - living  \
-		beings. They can, however, still struggle after an initial burst of inertia."
+		structures, machinery, vehicles, and, thanks to the fruitful efforts of its creators - living beings."
 	complexity = 0
 	prebuilt = TRUE
 	stat_required = CONSCIOUS
+
+/obj/item/mod/module/anomaly_locked/kinesis/admin
```
Comment (Time-Green) [inline code/modules/mod/modules/module_kinesis.dm:272]: Autodoc the type : )

**Review by Time-Green — CHANGES_REQUESTED** [review]

File: code/modules/mod/modules/module_kinesis.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/mod/modules/module_kinesis.dm:272]:
```suggestion
/// Admin suit version of kinesis. Can grab anything at any range, may enable phasing through walls.
/obj/item/mod/module/anomaly_locked/kinesis/admin
```

**Pull request merged**